DESIGN.md

```diff
@@ -103,8 +93,14 @@ ram_per_jruby = (512, 768, 1024) if total memory (4-7 GB, 8-16 GB, 16 GB+)
 ram_per_jruby_code_cache = 128
 ```
 
-PuppetServer jrubies are constrained based on both how many jrubies fit into unallocated memory and unallocated processors. PuppetServer memory is then set the amount of memory required for all jrubies.
+PuppetServer jrubies are constrained based on both how many jrubies fit into unallocated memory and unallocated processors (one per processor). PuppetServer memory is then set the amount of memory required for all jrubies.
 
+```
+possible jrubies by ram = (unreserved Ram) / (ram_per_jruby + ram_per_jruby_code_cache)
+#rjubies capped by (unreserved cpus) or the max of 24, whichever is less.
```

Comment:
```suggestion
jrubies = [possible_jrubies_by_ram, unreserved_cpus, MAX_JRUBIES].min
```

Reply:
I was trying to avoid writing it like code, as some reader may not be coders.